.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+<!--
+
+Make sure to run `msbuild RoslynAnalyzers.sln /t:pack` and `msbuild FxCopAnalyzeres.sln /t:pack` in the repository root when
+you have any changes that affect auto-generated files. Otherwise, the CI build will fail.
+
+-->
```

Comment:
Can you also add a pointer to GuidelinesForNewRules.md here?

Reply:
And explicitly call out documentation requirements for PRs adding new analyzers?